js/calculations/flowCalc.js

```diff
@@ -251,22 +251,7 @@ function calcAll() {
         } else {
             document.getElementById("staticPressure").innerText = height;
         }
-
-        if (roundTotal == "NaN" || roundTotal < 0) {
-            document.getElementById("totalPressure").innerText = "-";
-        }
-        if (roundPress == "NaN" || roundPress < 0) {
-            document.getElementById("pressureLoss").innerText = "-";
-        }
-        if (roundVel == "NaN" || roundVel < 0) {
-            document.getElementById("flowSpeed").innerText = "-";
-        }
-        if (height == "NaN") {
-            document.getElementById("staticPressure").innerText = "0";
-        } else {
-            document.getElementById("staticPressure").innerText = height;
-        }
-
+      
```

Comment:
Removing all the prevent-NaN code, is that relevant to fixed spaces? Is this a merge-gone-wrong perhaps?